spec/controllers/summary_controller_spec.rb

```diff
@@ -8,36 +8,8 @@
   before { stub_sso_data(prison) }
 
   context 'with 4 offenders' do
```

Comment:
it would have been nice not to replicate this - I have no idea why these 4 offenders are required for this test. The context says it all (i.e. its hopeless) - it used to say 'with 2 offenders' which was worse. A new clean test was probably called for.